### PR TITLE
Remove logging string interpolations from hot path

### PR DESCRIPTION
**Pull request by knopp**

### CODE REVIEW

**Comment by brian-superlist** [comment]
@matthew-carroll Ping on the follow the leader performance improvement

File: lib/src/follower.dart [hunk 1]
```diff
@@ -387,7 +387,8 @@ class RenderFollower extends RenderProxyBox {
       return;
     }
 
-    FtlLogs.follower.finest("Follower's LeaderLink reported a change: $_link. Requesting Follower child repaint.");
```
[the remark this answers is not in the document]
Comment (knopp) [reply lib/src/follower.dart:390]: The extra closures have been issue in the element visitor, but I haven't seen it in profile here. There are much less invocations I'd assume. 

It won't hurt but I don't think it's worth complicating the code except in tight loops (i.e. `_collectTransformForLayerChain`).

**Comment by matthew-carroll** [comment]
@knopp - Do you know if the widget state properties are now in stable? If so, can you update the handful of analysis errors?

**Comment by knopp** [comment]
> @knopp - Do you know if the widget state properties are now in stable? If so, can you update the handful of analysis errors?

Seems like it. I'll make another PR.

**Comment by knopp** [comment]
https://github.com/Flutter-Bounty-Hunters/follow_the_leader/pull/46

**Comment by matthew-carroll** [comment]
I merged the other PR. Please rebase this one.

**Review by matthew-carroll — APPROVED** [review]
LGTM

**Pull request merged**